src/style.css

```diff
@@ -1,3 +1,83 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
You are using the `*` selector to set styles for all elements. This can impact the performance of the page. It's better to set styles only for elements that require them.